perl/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM perl:latest
 
 RUN cpanm -n Twiggy Getopt::Args Dancer2
 
-COPY    server.pl /server.pl
+COPY perl/server.pl /server.pl
```

Comment:
Same here - there is no perl directory here

Reply:
Directory changes we should avoid.